lib/vtls/schannel.c

```diff
@@ -1574,7 +1594,332 @@ CURLcode Curl_schannel_random(unsigned char *entropy, size_t length)
   return CURLE_OK;
 }
 
-#ifdef _WIN32_WCE
+#ifdef HAVE_EXCLUSIVE_TRUST_MODE
+static CURLcode add_certs_to_store(HCERTSTORE trust_store,
```

Comment:
I'd be glad to attempt that change.

Any tips on where this new file would need to be added in the build system?

I've been only using the winbuild makefiles. AFAICT, those makefiles won't need to be updated to account for the new source file.

A quick grep shows `schannel.c` referenced in:

```
Makefile
Makefile.in
lib/Makefile
lib/Makefile.in
lib/Makefile.inc
packages/Symbian/group/libcurl.mmp
```

Will the Visual Studio project files need to be updated?